osu.Game/osu.Game.csproj

```diff
@@ -30,13 +30,13 @@
     <PackageReference Include="Microsoft.Data.Sqlite.Core" Version="7.0.2" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="7.0.0" />
     <PackageReference Include="Microsoft.Toolkit.HighPerformance" Version="7.1.2" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
This is not a mistake - https://github.com/ppy/osu-framework/pull/5805 forces this (otherwise build fails with "detected package downgrade").